examples/CMakeLists.txt

```diff
@@ -20,14 +20,18 @@ add_executable(provide_prefix ${PROVIDE_PREFIX_SOURCES} ${PUBLIC_HEADERS})
 add_executable(publisher ${PUBLISHER_SOURCES} ${PUBLIC_HEADERS})
 add_executable(subscriber ${SUBSCRIBER_SOURCES} ${PUBLIC_HEADERS})
 add_executable(wampcra ${WAMPCRA_SOURCES} ${PUBLIC_HEADERS})
-add_executable(uds ${UDS_SOURCES} ${PUBLIC_HEADERS})
 add_executable(websocket_callee ${WEBSOCKET_CALLEE_SOURCES} ${PUBLIC_HEADERS})
 
+if(NOT MSVC)
+  add_executable(uds ${UDS_SOURCES} ${PUBLIC_HEADERS})
+  target_link_libraries(uds examples_parameters)
+  set(crypto_librariees crypto ssl)
+endif()
```

Comment:
This skips crypto/ssl on Windows altogether. This is not what we want. Skipping UDS is fine of course, but not the crypto things.

Also, obviously you haven't tested this on Linux;) Because `crypto_librariees` has a typo in it ..